### ISSUE TITLE
Error: Multiple possible networks found, use a Network ID to be more specific

### ISSUE THREAD

**Issue by ipuc** [tmetsch/occi-os#89]
Hi,
I've installed OpenStack(Juno) with 5 compute node, 1 network node and 1 controller node, using CentOS 7.
I'm trying to use OCCI API in openstack and I'm using the branch https://github.com/sandlbn/occi-os/tree/stable/juno-neutron, that is:
pip install pyssf
in /usr/local/src -> git clone git://github.com/tmetsch/occi-os.git
in the occi-os directory -> python setup.py install
configure the api-paste.ini

In the project created there are two network: one external and one internal;
If I try to Create a new VM as written in https://wiki.openstack.org/wiki/Occi I have this error: 
"Multiple possible networks found, use a Network ID to be more specific"

I try to specific the Network ID adding:
-H 'Link: /network/id_network ; rel="http://schemas.ogf.org/occi/infrastructure#network"; category="http://schemas.ogf.org/occi/infrastructure#networkinterface"
replace the id_network with the my internal id_network, but the response is "Internal Server Error".
Thanks!

**Comment by tmetsch** [comment]
@sandlbn: can you check?

**Comment by sandlbn** [comment]
@ipuc : yes that's true, fix is on the way. This branch is under heavy development, and should be finally ready for production this Friday.

**Comment by ipuc** [comment]
thank you so much!!!
I'll try the updated branch when I see it online and I'll let you know
